src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -52,18 +56,48 @@ type SectionObject = {
     items: Item[];
 };
 
+// TODO: remove when Onyx data is available
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const mockedCardsList = {
+    test1: {
+        cardholder: {accountID: 1, lastName: 'Smith', firstName: 'Bob', displayName: 'Bob Smith', avatar: ''},
+        name: 'Test 1',
+        limit: 1000,
+        lastFourPAN: '1234',
+    },
```

Comment:
Similar to https://github.com/Expensify/App/pull/44469/files#r1661049859

`name` will be `card.nameValuePairs.cardTitle` and `limit` will be `card.nameValuePairs.unapprovedExpenseLimit`

But we can update that later